packages/run-protocol/src/vaultFactory/vaultDirector.js

```diff
@@ -33,15 +33,23 @@ import {
 const { details: X } = assert;
 
 /**
+ * @typedef {{
+ * collaterals: Brand[],
+ * penaltyPoolAllocation: AmountKeywordRecord,
+ * rewardPoolAllocation: AmountKeywordRecord,
+ * }} EconState
+ *
  * @typedef {Readonly<{
  * debtMint: ZCFMint<'nat'>,
- * collateralTypes: Store<Brand,VaultManager>,
+ * collateralTypes: Store<Brand,VaultManagerObject>,
  * electionManager: Instance,
  * directorParamManager: import('@agoric/governance/src/contractGovernance/typedParamManager').TypedParamManager<{VaultDirectorParams}>,
  * mintSeat: ZCFSeat,
  * penaltyPoolSeat: ZCFSeat,
  * rewardPoolSeat: ZCFSeat,
  * vaultParamManagers: Store<Brand, import('./params.js').VaultParamManager>,
+ * econNotifier: Notifier<EconState>
```

Comment:
"econ" isn't great but it may be OK.  We should use the same naming across all the contracts.